packages/cli/src/lifecycles/scaffold.js

```diff
@@ -23,49 +23,61 @@ const writePageComponentsFromTemplate = async (compilation) => {
     });
   };
 
-  const loadPageMeta = async (file, result, { metaComponent }) => {
-    return new Promise((resolve, reject) => {
-      try {
-        const { title, meta, route } = file;
-        const metadata = {
-          title,
-          meta,
-          route
-        };
+  const loadPageMeta = (file, result, { metaComponent }) => {
+    const { title, meta, route } = file;
+    const metadata = {
+      title,
+      meta,
+      route
+    };
 
-        result = result.replace(/METAIMPORT/, `import '${metaComponent}'`);
-        result = result.replace(/METADATA/, `const metadata = ${JSON.stringify(metadata)}`);
-        result = result.replace(/METAELEMENT/, '<eve-meta .attributes=\${metadata}></eve-meta>');
+    result = result.replace(/METAIMPORT/, `import '${metaComponent}'`);
+    result = result.replace(/METADATA/, `const metadata = ${JSON.stringify(metadata)}`);
+    result = result.replace(/METAELEMENT/, '<eve-meta .attributes=\${metadata}></eve-meta>');
 
-        resolve(result);
+    return result;
+  };
+
+  const writeComponentToFile = async (target, filename, result) => {
+    return new Promise(async(resolve, reject) => {
+      try {
+        await fs.ensureDir(target, { recursive: true });
+        await fs.writeFile(path.join(target, `${filename}.js`), result);
+        resolve();
       } catch (err) {
         reject(err);
```

Comment:
should we add `console.error` before all `reject`?  Seems coming out of this PR, more logging couldn't hurt (for us and users), at least until we are able to implement something like #15

Reply:
This specific thrown error is printed back in the main [index.js](https://github.com/ProjectEvergreen/greenwood/blob/master/packages/cli/index.js#L79) already.  If you follow all thrown errors they should be printed back at that same location, if not then of course we need the error output, but from what I can tell additional outputs within a catch clause elsewhere would be redundant.